packages/arb-token-bridge-ui/src/util/cctp/fetchCCTP.ts

```diff
@@ -0,0 +1,46 @@
+import { ChainId } from '../networks'
+import { getAPIBaseUrl, sanitizeQueryParams } from '..'
+import { Response } from '../../pages/api/cctp/[type]'
+
+export type FetchParams = {
+  walletAddress: string
+  l1ChainId: ChainId
+}
+
+async function fetchCCTP({
+  walletAddress,
```

Comment:
nit:

```suggestion
  address,
```

Reply:
I would keep `walletAddress` here actually. We have lots of different addresses in the hook (the different contract addresses, token address, etc), so I prefer the variable name to be clear